packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/server/translations/Messages_fr_FR.json

```diff
@@ -507,10 +512,7 @@
   "server_configuration_installation_gabarit_step1_delete_button_cancel": "Annuler la suppression",
   "server_configuration_installation_gabarit_step1_delete_button_valid": "Supprimer le gabarit",
   "server_configuration_installation_gabarit_step1_delete_fail": "Impossible de supprimer le gabarit <b>{{t0}}</b> ({{t1}})",
-  "server_configuration_installation_gabarit_step2_info": "Vous êtes sur le point de lancer l'installation d'un gabarit",
-  "server_configuration_installation_gabarit_step2_server": "Serveur :",
-  "server_configuration_installation_gabarit_step2_os": "Gabarit :",
-  "server_configuration_installation_gabarit_step2_hardwareRaid": "{{t0}} ({{t1}} disques)",
+  "server_configuration_installation_gabarit_step2_info": "Vous êtes sur le point de lancer l'installation du gabarit <b>{{t0}}</b> ({{t1}}) sur le serveur <b>{{t2}}</b>.",
```

Comment:
**nitpick:** Avoid HTML tags in translations. Instead include the tags in `translate-values`

Reply:
@anooparveti done